fenix/app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -1120,7 +1129,6 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
     private fun createBrowsingModeManager(initialMode: BrowsingMode): BrowsingModeManager {
         return DefaultBrowsingModeManager(initialMode, components.settings) { newMode ->
             updateSecureWindowFlags(newMode)
-            themeManager.currentTheme = newMode
```

Comment:
I'm wondering if this line could be contributing the the [UI test failure](https://console.firebase.google.com/project/moz-fenix/testlab/histories/bh.66b7091e15d53d45/matrices/5126815555857021441/details?stepId=bs.76202a4362d6cbe6&testCaseId=6&query=et.db9bdd1d90c33627&deeplink=true)?

Reply:
Good spot, yep I see now this shouldn't have been moved